ouroboros-network/src/Data/Sequence/Strict.hs

```diff
@@ -0,0 +1,221 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE PatternSynonyms #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE ViewPatterns #-}
+
+-- | Strict variants of 'Seq' operations.
+--
+-- TODO: Move this to @cardano-prelude@ as a new module, @Data.Sequence.Strict@.
+module Data.Sequence.Strict
+  ( StrictSeq (.., Empty, (:<|), (:|>))
+
+    -- * Construction
+    , empty
```

Comment:
Can you *de*dent the exports so that they commas line up with the opening and closing parentheses?